README.md

```diff
@@ -440,6 +440,135 @@ const subscriptionId = '<subscription_id>'; // player id
 await api.endLiveActivity('<app_id>', '<activity_id>', subscriptionId);
 ```
 
+## Users
+### Creating a OneSignal User
+```js
+const user = new OneSignal.User();
+
+const aliasLabel = '<alias_label>';
+const aliasId = '<alias_id>';
+const subscriptionToken = '<subscription_token>';
+
+user.identity = {
+    [aliasLabel]: aliasId,
+};
+
+user.subscriptions = [
+  {
+    type: "iOSPush",
+    token: subscriptionToken,
+  }
+];
+
+const createdUser = await api.createUser('<app_id>', user);
+assert(createdUser.identity!['onesignal_id'] != null);
+```
+
+### Getting a user by `onesignal_id`
+```js
+const fetchedUser = await api.fetchUser('<app_id>', "onesignal_id", createdUser.identity!['onesignal_id']);
+```
+
+### Getting a user by an alias
+```js
+const fetchedUser = await api.fetchUser('<app_id>', alias_label, alias_id);
+```
+
+### Updating a user
+```js
+const updateUserRequest: UpdateUserRequest = {
+    properties: {
+        language: 'fr'
+    }
+};
+
+const updatedUser = await api.updateUser('<app_id>', aliasLabel, aliasId, updateUserRequest);
+```
+
+### Deleting a user
+```js
+await api.deleteUser('<app_id>', aliasLabel, aliasId);
+```
+
+## Subscriptions
+### Creating a subscription for existing user
+```js
+const createSubscriptionRequestBody: CreateSubscriptionRequestBody = {
+    subscription: {
+        type: "AndroidPush",
```

Comment:
consider making an enum

Reply:
if not included in this release, at least link to a documented list of acceptable types from this section